tests/lib/formatters/unix.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @fileoverview Tests for unix-style formatter.
+ * @author Nicholas C. Zakas
```

Comment:
@oshi-shinobu you should put your name and copyright here, as you are the author.